globalize-versioning.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activerecord', '>= 3.2.0', '< 5'
   s.add_dependency 'activemodel', '>= 3.2.0', '< 5'
-  s.add_dependency 'globalize', '>= 3.0.4', '< 6'
+  s.add_dependency 'globalize', '5.1.0.beta2'
```

Comment:
Can we keep previous version restriction?

Reply:
Yeah, I will change it.